stable/aws-tigera-operator/templates/crs/custom-resources.yaml

```diff
@@ -0,0 +1,12 @@
+# This section includes base Calico installation configuration.
+# For more information, see: https://docs.projectcalico.org/v3.17/reference/installation/api#operator.tigera.io/v1.Installation
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+  annotations:
+    "helm.sh/hook": post-install
+spec:
```

Comment:
So actually since this file is hard coded instead of the [normal custom-resources.yaml](https://github.com/projectcalico/calico/blob/master/_includes/charts/tigera-operator/templates/crs/custom-resources.yaml), there isn't any chart configuration possible. Sorry I didn't notice that before.

So either you'll need to replace this file with what it originally had or the chart won't be configurable.